File: app/src/main/java/com/neome/java/api/meta/base/dto/StudioEntReportPlugin.java
// neome.ai API - do not change
//

package com.neome.java.api.meta.base.dto;

import com.neome.java.api.meta.base.Types.MetaIdVar;

import org.jetbrains.annotations.Nullable;

@SuppressWarnings("unused")
public class StudioEntReportPlugin extends StudioEntReport
{
  @Nullable
  public MetaIdVar inputFormMappingVarId;

  @Nullable
  public MetaIdVar outputFormMappingVarId;

  @Nullable
  public StudioDtoPluginApi pluginApi;
}

File: app/src/main/java/com/neome/api/meta/base/dto/StudioEntReportPlugin.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.Types.MetaIdVar
import com.neome.api.meta.base.dto.StudioDtoPluginApi
import com.neome.api.meta.base.dto.StudioEntReport

interface StudioEntReportPlugin : StudioEntReport
{
  val inputFormMappingVarId: MetaIdVar?
  val outputFormMappingVarId: MetaIdVar?
  val pluginApi: StudioDtoPluginApi?
}